test/AspNet.Security.OAuth.Providers.Tests/HubSpot/HubSpotTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using static AspNet.Security.OAuth.HubSpot.HubSpotAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.HubSpot;
+
+public class HubSpotTests : OAuthTests<HubSpotAuthenticationOptions>
+{
+    public HubSpotTests(ITestOutputHelper outputHelper)
+    {
+        OutputHelper = outputHelper;
+    }
+
+    public override string DefaultScheme => HubSpotAuthenticationDefaults.AuthenticationScheme;
+
+    protected internal override void RegisterAuthentication(AuthenticationBuilder builder)
+    {
+        builder.AddHubSpot(options => ConfigureDefaults(builder, options));
+    }
+
+    [Theory]
+    [InlineData(ClaimTypes.NameIdentifier, "dummy@dummymail.com")]
+    [InlineData(ClaimTypes.Email, "dummy@dummymail.com")]
+    [InlineData(Claims.HubId, "13371337")]
+    [InlineData(Claims.UserId, "123123")]
+    [InlineData(Claims.AppId, "696969")]
+    [InlineData(Claims.HubDomain, "dev-13371337.com")]
+
```

Comment:
It's preferred to use literals to decouple the tests from the implementation details.

```suggestion
    [InlineData(ClaimTypes.NameIdentifier, "dummy@dummymail.com")]
    [InlineData(ClaimTypes.Email, "dummy@dummymail.com")]
    [InlineData("urn:HubSpot:hub_id", "13371337")]
    [InlineData("urn:HubSpot:user_id", "123123")]
    [InlineData("urn:HubSpot:app_id", "696969")]
    [InlineData("urn:HubSpot:hub_domain", "dev-13371337.com")]
```